src/main/java/gregtech/api/recipes/RecipeBuilder.java

```diff
@@ -52,10 +55,14 @@ public class RecipeBuilder<R extends RecipeBuilder<R>> {
 
     protected final List<GTRecipeInput> inputs;
     protected final List<ItemStack> outputs;
-    protected final List<ChanceEntry> chancedOutputs;
+    protected final List<ChancedItemOutput> chancedOutputs;
```

Comment:
should this not be renamed to `chancedItemOutputs`?

Reply:
So far we've always been referring to item outputs as just `outputs`. I won't change from that convention in this PR since it would probably warrant a more widespread conversion.

Reply:
alright